main/modes/system/intro/introMode.c

```diff
@@ -226,7 +230,7 @@ static const tutorialStep_t buttonsSteps[] = {
             .buttons = PB_B, 
                     },
         .title = sonaTitle,
-        .detail = "This is a Sona. It's your personal avatar! Make your Sona by pressing B.",
+        .detail = "This is a Sona, your Swadge Avatar! You can make yours in the Sona Creator Mode. Press B to continue.",
```

Comment:
Had to add back "Press B to continue" otherwise it felt like you get stuck. Also adjusted Pinkie for one more line of text.